cicd/gitops/configuring-sso-on-argo-cd-using-dex.adoc

```diff
@@ -8,6 +8,8 @@ toc::[]
 
 After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, cluster administrators can use Argo CD to configure Single Sign-On (SSO).
 
+IMPORTANT: The `spec.dex` parameter in the ArgoCD CR is deprecated. In the future release of Red Hat Openshift GitOps v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. Consider using the `.spec.sso` parameter instead.
```

Comment:
"Red Hat OpenShift GitOps" should be {gitops-title} as far as I understand?

Also "is planned to be removed" is not very good English in my view. I would just write "will be removed". In the same phrase remove "the" - we do not actually know in which release it will be removed. 

To summarize, I'd make it: 

In a future release of {gitops-title} v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is likely to be removed.

Reply:
@mramendi Thanks for the feedback and suggestions. Good-catch! I am using the attribute and indefinite article as suggested.

Refer to https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts for deprecation notice writing guidelines and template. I am adhering to the same standards and prefer to retain the same across this PR. Thank you!